apps/frontend/tsconfig.app.json

```diff
@@ -18,5 +18,5 @@
     "**/*.spec.jsx",
     "**/*.test.jsx"
   ],
-  "include": ["**/*.js", "**/*.jsx", "**/*.ts", "**/*.tsx"]
+  "include": ["**/*.js", "**/*.jsx", "**/*.ts", "**/*.tsx", "src/app/components/CandidatePicker"]
```

Comment:
Is this necessary?

Reply:
Not entirely sure where this came from, might be something from nx.dev. Will attempt to remove it and see if something breaks. 😅